libs/wingc/src/lsp/rename_visitor.rs

```diff
@@ -168,6 +207,40 @@ impl<'a> Visit<'a> for RenameVisitor<'a> {
 				symbol: var_name.clone(),
 				references: vec![],
 			}),
+			//TODO: to be handled in a following PR, renaming interface fields is not supported yet
+			// StmtKind::Interface(c) => {
+			// for field in &c.methods {
+			// 	self.linked_symbols.push(LinkedSymbol {
+			// 		symbol: field.0.clone(),
+			// 		references: vec![],
+			// 	})
+			// }
+			// }
+			StmtKind::Struct(s) => {
+				for field in &s.fields {
+					self.linked_symbols.push(LinkedSymbol {
+						symbol: field.name.clone(),
+						references: vec![],
+					})
+				}
+			}
+			StmtKind::Class(c) => {
+				for (m, ..) in &c.methods {
+					if m.name == "new" {
+						continue;
+					}
+					self.linked_symbols.push(LinkedSymbol {
+						symbol: m.clone(),
+						references: vec![],
```

Comment:
if we try to lookup class methods it doesn't resolve to the interface decleration I think, so we need to find some other way to connect to them. I tried to use `c.implements` but it doesn't seems to work- and I don't want to add additional data structures if possible, what am I missing here? 
(I can resolve it on a following PR or in the current one, depends on the complexity of the solution )

Reply:
if it's possible I think that the logic of 
```
if class.implements[method]- add the base interface method as a linked symbol (if not exists)
    and reference extending interfaces, up to the class' method
if not- add it as a linked symbol
```
and on the interfaces add just the methods that don't exist yet

Reply:
> I tried to use c.implements but it doesn't seems to work

What part of it didn't work? I think you're right that class methods don't have a direct linkage to the interface they implement (if any), but it should be possible to lookup in the interfaces in c.implements.

Reply:
opened #6693 I'll try again there